test/docfx.Test/lib/GitUtilityTest.cs

```diff
@@ -22,9 +23,17 @@ public static class GitUtilityTest
             Assert.NotNull(repo);
 
             var pathToRepo = PathUtility.NormalizeFile(file);
-            var exe = await GitUtility.GetCommits(repo, pathToRepo);
-            var lib = GitUtility.GetCommits(repo, new List<string> { pathToRepo })[0].ToList();
-            Assert.Equal(JsonConvert.SerializeObject(exe), JsonConvert.SerializeObject(lib));
+
+            try
+            {
+                var exe = await GitUtility.GetCommits(repo, pathToRepo);
+                var lib = GitUtility.GetCommits(repo, new List<string> { pathToRepo })[0].ToList();
+                Assert.Equal(JsonConvert.SerializeObject(exe), JsonConvert.SerializeObject(lib));
+            }
+            catch (NotSupportedException ex)
```

Comment:
that means we never run this test successfully in CI? because CI is using shallow clone?

Reply:
only travis, on linux

Reply:
git clone -b v3 --single-branch?

---
In reply to: [188843300](https://github.com/dotnet/docfx/pull/2781#discussion_r188843300) [](ancestors = 188843300)

Reply:
the git clone command is generated/managed by travis